tests/fast/BulkDumping.toml

```diff
@@ -1,6 +1,4 @@
 [configuration]
-storageEngineExcludeTypes = [5] #FIXME: remove after allowing to do bulkloading with fetchKey and shardedrocksdb
```

Comment:
Oh, we still want to exclude sharded rocksdb because the ss metadata change for the sharded rocksdb engine is still working in progress. Currently, we do not support sharded rocksdb for bulk loading.

Reply:
Ok. Let me fix.